src/app/api/actions/ajouterUneNoteDeContexteAction.ts

```diff
@@ -0,0 +1,42 @@
+import { revalidatePath } from 'next/cache'
+import { z } from 'zod'
+
+import prisma from '../../../../prisma/prismaClient'
+import { PrismaGouvernanceRepository } from '@/gateways/PrismaGouvernanceRepository'
+import { PrismaUtilisateurRepository } from '@/gateways/PrismaUtilisateurRepository'
+import { ResultAsync } from '@/use-cases/CommandHandler'
+import { AjouterNoteDeContexteAGouvernance } from '@/use-cases/commands/AjouterNoteDeContexteAGouvernance'
+
+export async function ajouterUneNoteDeContexteAction(
+  actionParam: ActionParams
+): ResultAsync<ReadonlyArray<string>> {
+  const validationResult = validator.safeParse(actionParam)
+
+  if (validationResult.error) {
+    return validationResult.error.issues.map(({ message }) => message)
+  }
+
+  const ajouterNoteDeContexteAGouvernance = new AjouterNoteDeContexteAGouvernance(
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-argument, @typescript-eslint/no-explicit-any
+    new PrismaGouvernanceRepository(prisma.gouvernanceRecord) as any,
+    new PrismaUtilisateurRepository(prisma.utilisateurRecord),
+    new Date()
+  )
+  const result = await ajouterNoteDeContexteAGouvernance.execute(actionParam)
+  revalidatePath(validationResult.data.path)
+  return [result]
+}
+
+type ActionParams = Readonly<{
+  contenu: string
+  uidGouvernance: string
+  uidUtilisateurCourant: string
+  path: string
+}>
+
+const validator = z.object({
+  contenu: z.string().min(1, { message: 'Le contenu doit être renseigné' }),
+  path: z.string().min(1, { message: 'Le chemin doit être renseigné' }),
+  uidGouvernance: z.string().min(1, { message: 'L‘identifiant de la gouvernance doit être renseigné' }),
+  uidUtilisateurCourant: z.string().min(1, { message: 'L‘identifiant de l‘utilisateur courant doit être renseigné' }),
```

Comment:
```suggestion
  path: z.string().min(1, { message: 'Le chemin doit être renseigné' }),
}),
```
C'est un truc qu'on a oublié de partagé mais à partir du moment où c'est tester dans les use-cases/domain alors plus la peine de le faire ici ce qui simplifie grandement !
On garde juste la validation technique donc juste le `path`